pages/osx/codesign.md

```diff
@@ -0,0 +1,11 @@
+# codesign
+
+> Create and manipulate code signatures for macOS.
+
+- Sign for an application with your certificate:
```

Comment:
Can we remove the "for" ? I believe this command signs an application with a certificate right ? In that case, the "for" is not needed.